manifest/manifest.go

```diff
@@ -14,8 +14,10 @@ import (
 )
 
 type FileManifest struct{
-    InputFile map[string]string `json:"inputfile"`
-    OutputFile map[string]string `json:"outputfile"`
+    InputFile []string `json:"inputfile"`
+    InputFileHash []string `json:"inputfile_hash"`
+    OutputFile []string `json:"outputfile"`
+    OutputFileHash []string `json:"outputfile_hash"`
 }
 
```

Comment:
Is the manifest file now a list of input files followed by a list of input file hashes, followed by a list of output files followed by a list of outputfilehashes ?
They will be come difficult to read.
Wondering if tit should be a map, but an ordered map? 
OR
list of of tuples(file name, hash)